src/index.js

```diff
@@ -29,11 +29,10 @@ const TIMEOUT = 30_000
 export default {
   /** @type {import('@web3-storage/gateway-lib').Handler<import('@web3-storage/gateway-lib').Context, import('./bindings').Environment>} */
   async fetch (request, env, ctx) {
-    console.log(request.method, request.url)
     const middleware = composeMiddleware(
       withCdnCache,
       withCorsHeaders,
-      withErrorHandler,
```

Comment:
@alanshaw Given secret is needed to setup Sentry client to then use it in error it made more sense for me to be repo specific. We can also require it in the context in gateway-lib and with that use it via `withErrorHandler` from there, having a new middleware to setup sentry in context here.

What would you prefer?